src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -2633,381 +2642,7 @@
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\BackGesture\BackGesture_NavigationCommand.xaml.cs">
-      <DependentUpon>BackGesture_NavigationCommand.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\BackGesture\BackGesture_Normal.xaml.cs">
-      <DependentUpon>BackGesture_Normal.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_BackGesture.xaml.cs">
-      <DependentUpon>CommandBar_BackGesture.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Content.xaml.cs">
-      <DependentUpon>CommandBar_Content.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Dynamic.xaml.cs">
-      <DependentUpon>CommandBar_Dynamic.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Examples.xaml.cs">
-      <DependentUpon>CommandBar_Examples.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Extensions.xaml.cs">
-      <DependentUpon>CommandBar_Extensions.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Flyout.xaml.cs">
-      <DependentUpon>CommandBar_Flyout.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_LongTitle.xaml.cs">
-      <DependentUpon>CommandBar_LongTitle.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Native.xaml.cs">
-      <DependentUpon>CommandBar_Native.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Padding.xaml.cs">
-      <DependentUpon>CommandBar_Padding.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_Uppercase_Resource.xaml.cs">
-      <DependentUpon>CommandBar_Uppercase_Resource.xaml</DependentUpon>
-    </Compile>
-    <Compile Update="C:\s\nv.github\uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\CommandBar\CommandBar_With_Long_Sentences.xaml.cs">
-      <DependentUpon>CommandBar_With_Long_Sentences.xaml</DependentUpon>
-    </Compile>
-  </ItemGroup>
```

Comment:
Removed a lot of file nesting declarations...

Reply:
Removed a lot of invalid absolute path ;)